wgsl/index.bs

```diff
@@ -2468,7 +2468,7 @@ statement
   | continue_statement SEMICOLON
   | DISCARD SEMICOLON
   | assignment_statement SEMICOLON
-  | body_statement
+  | body_statement?
```

Comment:
is this even correct grammar? if `statement` is allowed to be empty, how would the grammar know what `statement*` is?

Reply:
It means `statement*` is an empty list. So, `if (a) {}` would fall into that as there are no statements.

Reply:
in what context can the statement be empty?

Reply:
would it? the body of the `if` statement is `statement*`, so it can be zero or more statements, so empty would be that zero already

Reply:
It looks like the one place it would matter is in the case body at the moment as it uses statement directly. But, that should be possible to change to statements and have it work the same.

Maybe this is just a parser bug in tint with the way I handled statements. Will close this and look at fixing over there.